File: src/main/java/com/projectswg/holocore/services/gameplay/structures/StructuresManager.java
package com.projectswg.holocore.services.gameplay.structures;

import com.projectswg.holocore.services.gameplay.structures.housing.HousingManager;
import me.joshlarson.jlcommon.control.Manager;
import me.joshlarson.jlcommon.control.ManagerStructure;

@ManagerStructure(children = {
		HousingManager.class
})
public class StructuresManager extends Manager {
	
	public StructuresManager() {
		
	}
	
}

File: src/main/java/com/projectswg/holocore/services/gameplay/structures/StructuresManager.kt
package com.projectswg.holocore.services.gameplay.structures

import com.projectswg.holocore.services.gameplay.structures.housing.HousingManager
import com.projectswg.holocore.services.gameplay.structures.terminals.ElevatorService
import me.joshlarson.jlcommon.control.Manager
import me.joshlarson.jlcommon.control.ManagerStructure

@ManagerStructure(children = [
	HousingManager::class,
	StructureService::class,
	ElevatorService::class
])
class StructuresManager : Manager()
